src/schemas/delete-entity-operation-config.schema.json

```diff
@@ -15,6 +15,14 @@
     },
     "dataJsonPath": {
       "type": "string"
+    },
+    "responseModel": {
+      "type": "object",
+      "properties": {
+      }
+    },
+    "delay": {
+      "type": "boolean"
```

Comment:
the delay should be a number and we need to have a default for it and this number just changes the delay for this one...